tasks.php

```diff
@@ -1081,12 +1088,12 @@ function select_and_list_tasks($sql_condition)
         }
     }
     else {
-        echo "<tr bgcolor='#ffffff'><td colspan='7'><center>No tasks found!</center></td></tr>";
+        echo "<tr><td colspan='7'><center>No tasks found!</center></td></tr>";
     }
     echo "</table><br>\n";
     // if 2 tasks or more found, display the number of reported tasks
     if (@mysqli_num_rows($sql_result) > 1) {
-        echo "<p>" . @mysqli_num_rows($sql_result) . " tasks listed.</p>";
+        echo "<small class='task'>" . @mysqli_num_rows($sql_result) . " tasks listed.</small>";
     }
```

Comment:
I'm probably missing something here -- but I'm seeing a switch from a block element to an inline element. Is this already dealt with elsewhere?

Reply:
I agree it's a change, Sharon. But the text string is inside a div element and that's all that should be needed, technically, I believe. The W3C HTML validator doesn't object to it, at least.

Reply:
Thanks, Walt, that's what I was missing. Yes, a div element is plenty.